### ISSUE TITLE
What is the Lifeline problem?

### ISSUE THREAD

**Issue by zvodd** [smallcluster/GDST#1]
Do you have any links to prior research. 
Googling doesn't yield relevant results.

**Comment by smallcluster** [comment]
You're right, I also got some trouble getting papers on the subject, if I remember correctly it's a problem exposed only in one paper where drones are used to assure a line of communication (drones have a lot of limitations like vision range, movement, memory, etc) between a fixed point and a moving point. 

Internally, we referred to it as the "Lifeline problem" as one of its applications were maintaining a communication between a team of first responder and their base in a large hazardous environment (where only short range communication is possible). 

I kept a bibliography somewhere, but some papers weren't ready for publishing at the time (not sharable), I will add what I can to the README with a more precise description of this informal "Lifeline problem".
